static/app/utils/replays/hooks/useRageDeadSelectors.tsx

```diff
@@ -0,0 +1,25 @@
+import {useApiQuery} from 'sentry/utils/queryClient';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export function useRageDeadSelectors(limit: number, sort: string) {
```

Comment:
not sure if passing in these parameters is the best thing to do here, but we need to be able to choose whether to return top selectors for rage clicks or top selectors for dead clicks